jest.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+    // The root of your source code, typically /src
```

Comment:
# To implement
- Unit tests components asap
- Unit tests on screens
- Unit tests on the global state